hack/e2e-test.sh

```diff
@@ -34,15 +34,18 @@ function cleanup {
         kubectl logs -n kube-system kube-controller-manager-kind-control-plane > $ARTIFACTS/kube-controller-manager.log || true
         kubectl logs -n kueue-system deployment/kueue-controller-manager > $ARTIFACTS/kueue-controller-manager.log || true
         kubectl describe pods -n kueue-system > $ARTIFACTS/kueue-system-pods.log || true
-        $KIND delete cluster --name $KIND_CLUSTER_NAME || { echo "You need to run make kind-image-build before this script"; exit -1; }
+        $KIND delete cluster --name $KIND_CLUSTER_NAME -v 5
     fi
     (cd config/components/manager && $KUSTOMIZE edit set image controller=gcr.io/k8s-staging-kueue/kueue:main)
 }
 
 function startup {
     if [ $CREATE_KIND_CLUSTER == 'true' ]
     then
-        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION --config $SOURCE_DIR/kind-cluster.yaml --wait 1m
+        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION --config $SOURCE_DIR/kind-cluster.yaml --wait 1m -v 5
```

Comment:
Can we start with something like v2?

Reply:
let's output to another file.... just to reduce clutter from the main logs

Reply:
> let's output to another file.... just to reduce clutter from the main logs

+1

Otherwise LGTM

Reply:
done, redirect to file, `cat` the file if create fails